docs/usage.md

```diff
@@ -0,0 +1,192 @@
+# AEMM Usage
+This page serves as documentation for AEMM's advanced use cases and behavior.
+
+
+## Commands
+AEMM's supported commands (`spotitn`, `scheduledevents`) are viewed using `--help`:
+```
+$ amazon-ec2-metadata-mock --help
+
+...
+Available Commands:
+  help            Help about any command
+  scheduledevents Mock EC2 Scheduled Events
+  spotitn         Mock EC2 Spot interruption notice
+
+...
+```
+commands are designed as follows:
+* Run independently from other commands
+  * i.e. when AEMM is started with `scheduledevents` subcommand, `spotitn` routes will **NOT** be available and vice-versa 
+* Local flag availability so that commands can be configured directly via CLI parameters
+    * With validation checks
+* Contain additional `--help` documentation
+* Default values sourced from code
+
+Metadata categories that are always available, irrespective of the CLI command run are referred to as **static metadata**.
+
+### Spot Interruptions
+To view the available local flags for the Spot Interruption command use `spotitn --help`:
```

Comment:
`available flags` as all flags are displayed

Reply:
👍🏼 I'll make the update